### PR TITLE
Draft of frontend code [WIP]

### PR DESCRIPTION
**Pull request by ashx3s**
- Mobile oriented navigation (should have a separate nav for desktop)
- Footer in process
- main body content is primarily placeholder for the time being

### CODE REVIEW

File: src/fedora_easyfix/static/css3/USAGE [hunk 1]
```diff
@@ -1 +0,0 @@
-Add all CSS3 assets here
```
Comment (t0xic0der) [inline src/fedora_easyfix/static/css3/USAGE:1]: I would ask you to revert this file back as it acts as in-code documentation.

File: src/fedora_easyfix/templates/mainPage.html [hunk 2]
```diff
@@ -0,0 +1,237 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <!-- For bootstrap-->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
```
Comment (t0xic0der) [inline src/fedora_easyfix/templates/mainPage.html:7]: Please stick to using Bootstrap components as much as possible.

File: src/fedora_easyfix/templates/mainPage.html [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.1/dist/js/bootstrap.bundle.min.js"
+        integrity="sha384-/bQdsTh/da6pkI1MST/rWKFNjaCP5gBSY4sEBT38Q/9RBh9AH40zEOg7Hlq2THRZ"
+        crossorigin="anonymous"></script>
+
+    <meta charset="UTF-8">
+    <title>Fedora Easyfix</title>
+</head>
+
+<body>
+    <nav class="navbar navbar-light bg-light">
```
Comment (t0xic0der) [inline src/fedora_easyfix/templates/mainPage.html:18]: Please ensure that the navbar sticks to the top even when the viewport has been scrolled down.

File: src/fedora_easyfix/templates/mainPage.html [hunk 4]
```diff
@@ -0,0 +1,237 @@
+<!DOCTYPE html>
```
Comment (t0xic0der) [inline src/fedora_easyfix/templates/mainPage.html:1]: Please revert the name back to `mainpage.html`.

**Review by gridhead — CHANGES_REQUESTED** [review]
Apart from the suggested changes, please revert back the directory names as they were before.

**Comment by t0xic0der** [comment]
Some cents regarding the commit messages too
1. Make sure that your commit messages do not exceed 50 characters.
2. If more information is required, use the body up to 72 characters.
3. Use the message to specifically describe the outcome of the commit.

Do amend the commits before requesting another review.

File: src/fedora_easyfix/templates/mainPage.html (hunk 2 above)
Comment (lilyx13) [reply src/fedora_easyfix/templates/mainPage.html]: With this being tagged onto the CDN, do you want bootstrap to be installed as a dependency in the repo? I'm not sure how you mean that here.

**Pull request merged**